api/resolvers/item.js

```diff
@@ -415,7 +415,7 @@ export default {
               ORDER BY "Item".created_at DESC
```

Comment:
You'll want to order the inner query too so that the LIMIT doesn't cut out the item. I'll make the change myself on review but the way this query monster works is we have LIMIT'd subquery so we can efficiently join the result0 with a bunch things in result1. After the join result0 can get out of order so we have to reorder it.